docs/_getting-started-linux-android.md

```diff
@@ -47,23 +47,22 @@ Next, select the "SDK Tools" tab and check the box next to "Show Package Details
 
 Finally, click "Apply" to download and install the Android SDK and related build tools.
 
-<h4>3. Configure the ANDROID_HOME environment variable</h4>
+<h4>3. Configure the ANDROID_SDK_ROOT environment variable</h4>
 
 The React Native tools require some environment variables to be set up in order to build apps with native code.
 
 Add the following lines to your `$HOME/.bash_profile` or `$HOME/.bashrc` (if you are using `zsh` then `~/.zprofile` or `~/.zshrc`) config file:
 
 ```shell
-export ANDROID_HOME=$HOME/Android/Sdk
-export PATH=$PATH:$ANDROID_HOME/emulator
-export PATH=$PATH:$ANDROID_HOME/tools
-export PATH=$PATH:$ANDROID_HOME/tools/bin
-export PATH=$PATH:$ANDROID_HOME/platform-tools
+export ANDROID_SDK_ROOT=$HOME/Library/Android/sdk
+export PATH=$PATH:$ANDROID_SDK_ROOT/emulator
+export PATH=$PATH:$ANDROID_SDK_ROOT/cmdline-tools/latest/bin
```

Comment:
Why this?

Reply:
My understanding is that the `tools/bin` path has been deprecated since v4 of the command line tools. If I try to use the `sdkmanager` binary, from the old path, with JDK v11+ I get the following with literally any argument:

<img width="304" alt="Screenshot 2022-03-03 at 18 09 16" src="https://user-images.githubusercontent.com/25727106/156626096-30678acb-7459-44f4-9748-22c94879535a.png">

However if I use sdkmanager from `/cmdline-tools/latest/bin`:

<img width="514" alt="Screenshot 2022-03-03 at 18 04 25" src="https://user-images.githubusercontent.com/25727106/156626219-da63691f-e60b-4ccb-90ea-bd33fc0ca9bb.png">

In similar fashion if I use binaries from `tools`. They either fail, work unexpectedly, or even state they are deprecated:

<img width="304" alt="Screenshot 2022-03-03 at 18 09 16" src="https://user-images.githubusercontent.com/25727106/156626319-7365a998-daa9-44f1-8aa6-c31b6c8e269d.png">

Reply:
My question was related to the `cmdline-tools/latest/bin` path which I don't seem to have in my SDK setup

Reply:
This could be bad assumption on my part but assumed that the path for linux would be the same as macos. Assuming you are on a linux machine?

Happy to revert the path change for the linux docs and just leave the env var change if you agree that is the best approach?

Reply:
As a suggestion maybe we should make clear that the path is determined by version. 

i.e. 

`cmdline-tools/[VERSION]/bin`

And then example `latest` as imagine that will be the case for most people who are installing android studio for the first time?

Reply:
I've pushed a change to add a comment explaining the versioning. Let us know what you think and/or if you have better suggestion to present this.

Reply:
I don't have SDK Command-line Tools installed, and I also believe this should not be a hard requirement for the React Native setup. I believe we should be fine by removing it entirely. Which tool are we using from there?

Reply:
I've removed reference to command line path for linux getting started docs.